### ISSUE TITLE
Multiple language support

### ISSUE THREAD

**Issue by Frijol** [tessel/t2-start#5]
How do we show a Rust getting started, JS getting started?

Somewhat relatedly, what about foreign languages – is this something we want to support at all?

(right now we have: 
![screen shot 2015-06-13 at 11 56 13 am](https://cloud.githubusercontent.com/assets/454690/8145618/39fcc76e-11c3-11e5-83c3-d03a1dcd1316.png)
)

**Comment by johnnyman727** [comment]
We'll just need CLI documentation and code examples for each supported language. The user should be able to select a language they're interested on any start page and have that selection saved on the subsequent pages.

**Comment by Frijol** [comment]
Yes, that's a bit tautological. I'm more hoping this issue can be a discussion of how we want to see that done– does it make sense to try to do this the same way current tessel.io/start has English vs. Japanese, in the menu bar, or are there different needs?

**Comment by johnnyman727** [comment]
Well, since you're looking for more a graphic design question, I'll let someone else suggest a specific implementation.

In terms of spoken language, I was always surprised that we didn't layout the website like www.tessel.io/jp/start or www.tessel.io/es/start.

**Comment by tcr** [comment]
We removed the Japanese translation from the Github docs while they are being rewritten. This is a good opportunity to consider how to scale this to many languages.

**Comment by Frijol** [comment]
I propose that we handle localization and multiple programming languages in a manner similar to Gitbooks: https://toolchain.gitbook.com/languages.html
